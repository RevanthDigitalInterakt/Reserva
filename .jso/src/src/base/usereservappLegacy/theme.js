  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.theme = undefined;
  var tokens = _interopRequireWildcard(_$$_REQUIRE(_dependencyMap[0]));
  function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(e) { return e ? t : r; })(e); }
  function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
  var theme = exports.theme = {
    space: {
      quarck: 4,
      nano: 8,
      micro: 12,
      xxxs: 16,
      xxs: 24,
      xs: 32,
      sm: 40,
      md: 48,
      xl: 64,
      xxl: 80
    },
    shadows: {
      bottomBarShadow: '0px -3px 2px rgba(0,0,0,0.1)',
      topBarShadow: '0px 3px 2px rgba(0,0,0,0.1)'
    },
    colors: {
      preto: tokens.preto,
      vermelhoFechadoRSV: tokens.vermelhoFechadoRSV,
      marinho: tokens.marinho,
      neutroFrio2: tokens.neutroFrio2,
      neutroQuente2: tokens.neutroQuente2,
      vermelhoRSV: tokens.vermelhoRSV,
      verde: tokens.verde,
      neutroQuente1: tokens.neutroQuente1,
      offWhite: tokens.offWhite,
      offWhite15: tokens.offWhite15,
      whiteSecondary: tokens.whiteSecondary,
      neutroFrio1: tokens.neutroFrio1,
      leve: tokens.leve,
      vermelhoAlerta: tokens.vermelhoAlerta,
      verdeSucesso: tokens.verdeSucesso,
      amareloAtencao: tokens.amareloAtencao,
      dropDownBorderColor: tokens.dropDownBorderColor,
      white: tokens.white,
      fullBlack: tokens.fullBlack,
      divider: tokens.divider,
      backgoundInput: tokens.backgoundInput,
      backgroundApp: tokens.backgroundApp,
      transparente: 'transparent',
      modalBackDropColor: tokens.modalBackDropColor,
      borderButton: tokens.borderButton,
      dividerCounter: tokens.dividerCounter,
      backgoundDivider: tokens.backgoundDivider,
      progressTextColor: tokens.progressTextColor,
      backgroundMenuOpened: tokens.backgroundMenuOpened,
      searchBarTextColor: tokens.searchBarTextColor,
      borderColorProduct: tokens.borderColorProduct
    },
    fonts: {
      nunitoBlack: 'Nunito-Black',
      nunitoBlackItalic: 'Nunito-BlackItalic',
      nunitoBold: 'Nunito-Bold',
      nunitoBoldItalic: 'Nunito-BoldItalic',
      nunitoExtraBold: 'Nunito-ExtraBold',
      nunitoExtraBoldItalic: 'Nunito-ExtraBoldItalic',
      nunitoExtraLight: 'Nunito-ExtraLight',
      nunitoExtraLightItalic: 'Nunito-ExtraLightItalic',
      nunitoItalic: 'Nunito-Italic',
      nunitoLight: 'Nunito-Light',
      nunitoLightItalic: 'Nunito-LightItalic',
      nunitoRegular: 'Nunito-Regular',
      nunitoSemiBold: 'Nunito-SemiBold',
      nunitoSemiBoldItalic: 'Nunito-SemiBoldItalic',
      reservaDisplayRegular: 'ReservaDisplay-Regular',
      reservaSerifBlack: 'ReservaSerif-Black',
      reservaSerifBlackItalic: 'ReservaSerif-BlackItalic',
      reservaSerifBold: 'ReservaSerif-Bold',
      reservaSerifBoldItalic: 'ReservaSerif-BoldItalic',
      reservaSerifItalic: 'ReservaSerif-Italic',
      reservaSerifLight: 'ReservaSerif-Light',
      reservaSerifLightItalic: 'ReservaSerif-LightItalic',
      reservaSerifMedium: 'ReservaSerif-Medium',
      reservaSerifMediumItalic: 'ReservaSerif-MediumItalic',
      reservaSerifRegular: 'ReservaSerif-Regular',
      reservaSerifThin: 'ReservaSerif-Thin',
      reservaSerifThinItalic: 'ReservaSerif-ThinItalic',
      reservaSansBlack: 'ReservaSans-Black',
      reservaSansBlackItalic: 'ReservaSans-BlackItalic',
      reservaSansBold: 'ReservaSans-Bold',
      reservaSansBoldItalic: 'ReservaSans-BoldItalic',
      reservaSansItalic: 'ReservaSans-Italic',
      reservaSansLight: 'ReservaSans-Light',
      reservaSansLightItalic: 'ReservaSans-LightItalic',
      reservaSansMedium: 'ReservaSans-Medium',
      reservaSansMediumItalic: 'ReservaSans-MediumItalic',
      reservaSansRegular: 'ReservaSans-Regular',
      reservaSansThin: 'ReservaSans-Thin',
      reservaSansThinItalic: 'ReservaSans-ThinItalic'
    },
    fontWeights: {
      light: 300,
      normal: 400,
      medium: 500,
      SemiBold: 600,
      bold: 700,
      black: 900
    },
    fontSizes: [4, 8, 12, 14, 16, 18, 20, 24, 27, 32, 48],
    radii: {
      pico: 2,
      nano: 5,
      xxxs: 10,
      xs: 15,
      sm: 20,
      pill: 500,
      infinity: 999999
    },
    borderWidths: {
      none: tokens.borderWidthNone,
      hairline: tokens.borderWidthHairline,
      thin: tokens.borderWidthThin,
      thick: tokens.borderWidthThick,
      heavy: tokens.borderWidthHeavy,
      borderDiscountTag: tokens.borderDiscountTag,
      stepper: 3
    }
  };
