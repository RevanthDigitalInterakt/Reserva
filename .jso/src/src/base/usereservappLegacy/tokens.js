  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.whiteSecondary = exports.white = exports.vermelhoRSV = exports.vermelhoFechadoRSV = exports.vermelhoAlerta = exports.verdeSucesso = exports.verde = exports.searchBarTextColor = exports.progressTextColor = exports.preto = exports.offWhite15 = exports.offWhite = exports.neutroQuente2 = exports.neutroQuente1 = exports.neutroFrio2 = exports.neutroFrio1 = exports.modalBackDropColor = exports.marinho = exports.leve = exports.fullBlack = exports.dropDownBorderColor = exports.dividerCounter = exports.divider = exports.borderWidthThin = exports.borderWidthThick = exports.borderWidthNone = exports.borderWidthHeavy = exports.borderWidthHairline = exports.borderDiscountTag = exports.borderColorProduct = exports.borderButton = exports.backgroundMenuOpened = exports.backgroundApp = exports.backgoundInput = exports.backgoundDivider = exports.amareloAtencao = undefined;
  var white = exports.white = '#FFF';
  var fullBlack = exports.fullBlack = '#000';

  // cores primarias
  var preto = exports.preto = '#333333';
  var vermelhoFechadoRSV = exports.vermelhoFechadoRSV = '#5A0000';
  var marinho = exports.marinho = '#13294B';
  var neutroFrio2 = exports.neutroFrio2 = '#8A8C8E';
  var neutroQuente2 = exports.neutroQuente2 = '#B1ACA8';
  var divider = exports.divider = '#D8D9DA';
  var dividerCounter = exports.dividerCounter = '#EAEAEA';
  var progressTextColor = exports.progressTextColor = '#979797';
  var searchBarTextColor = exports.searchBarTextColor = '#666666';

  // cores secundarias
  var vermelhoRSV = exports.vermelhoRSV = '#A00000';
  var verde = exports.verde = '#22372B';
  var neutroQuente1 = exports.neutroQuente1 = '#D7CFCA';
  var offWhite = exports.offWhite = '#D6D1C4';
  var offWhite15 = exports.offWhite15 = '#F9F8F6';
  var whiteSecondary = exports.whiteSecondary = '#FAFAFA';
  var neutroFrio1 = exports.neutroFrio1 = '#D8D9DA';
  var leve = exports.leve = '#F9E1DD';

  // cores do sistema
  var vermelhoAlerta = exports.vermelhoAlerta = '#EF1E1E';
  var verdeSucesso = exports.verdeSucesso = '#38A238';
  var amareloAtencao = exports.amareloAtencao = '#FEA528';
  var dropDownBorderColor = exports.dropDownBorderColor = '#DDDDDD';

  // larguras de borda
  var borderWidthNone = exports.borderWidthNone = 0;
  var borderWidthHairline = exports.borderWidthHairline = 1;
  var borderWidthThin = exports.borderWidthThin = 2;
  var borderWidthThick = exports.borderWidthThick = 4;
  var borderWidthHeavy = exports.borderWidthHeavy = 8;
  var borderDiscountTag = exports.borderDiscountTag = 12;

  // cores de borda
  var borderButton = exports.borderButton = '#333333';
  var borderColorProduct = exports.borderColorProduct = '#757575';

  // backgrounds
  var backgroundMenuOpened = exports.backgroundMenuOpened = '#F1F1F1';
  var backgroundApp = exports.backgroundApp = '#F9F9F9';
  var modalBackDropColor = exports.modalBackDropColor = '#AFAFAF';
  var backgoundInput = exports.backgoundInput = '#F0F0F0';
  var backgoundDivider = exports.backgoundDivider = '#F5F5F5';
