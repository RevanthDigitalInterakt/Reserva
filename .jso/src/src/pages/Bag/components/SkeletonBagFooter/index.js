  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.default = SkeletonBagFooter;
  var _react = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[1]));
  function SkeletonBagFooter() {
    return /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsx)(_$$_REQUIRE(_dependencyMap[3]).Box, {
      px: "xxs",
      testID: "com.usereserva:id/skeletonBagFooter_bag",
      children: /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsxs)(_$$_REQUIRE(_dependencyMap[3]).Box, {
        children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsxs)(_$$_REQUIRE(_dependencyMap[3]).Box, {
          flexDirection: "row",
          justifyContent: "space-between",
          marginTop: "micro",
          children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsxs)(_$$_REQUIRE(_dependencyMap[3]).Box, {
            children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsx)(_$$_REQUIRE(_dependencyMap[3]).Box, {
              bg: "neutroFrio1",
              width: 63,
              height: 22
            }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsx)(_$$_REQUIRE(_dependencyMap[3]).Box, {
              bg: "neutroFrio1",
              width: 95,
              height: 22,
              marginTop: "nano"
            })]
          }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsxs)(_$$_REQUIRE(_dependencyMap[3]).Box, {
            alignItems: "flex-end",
            children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsx)(_$$_REQUIRE(_dependencyMap[3]).Box, {
              bg: "neutroFrio1",
              width: 63,
              height: 22
            }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsx)(_$$_REQUIRE(_dependencyMap[3]).Box, {
              bg: "neutroFrio1",
              width: 95,
              height: 22,
              marginTop: "nano"
            })]
          })]
        }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[2]).jsx)(_$$_REQUIRE(_dependencyMap[3]).Box, {
          bg: "neutroFrio1",
          width: "100%",
          height: 50,
          marginTop: "micro"
        })]
      })
    });
  }
