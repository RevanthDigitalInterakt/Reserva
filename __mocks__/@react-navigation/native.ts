const navigate = jest.fn();

const useNavigation = () => ({
  navigate,
});

export {
  useNavigation,
};
