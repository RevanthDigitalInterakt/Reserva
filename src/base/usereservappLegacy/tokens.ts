export const white = '#FFF';
export const fullBlack = '#000';

// cores primarias
export const preto = '#333333';
export const vermelhoFechadoRSV = '#5A0000';
export const marinho = '#13294B';
export const neutroFrio2 = '#8A8C8E';
export const neutroQuente2 = '#B1ACA8';
export const divider = '#D8D9DA';
export const dividerCounter = '#EAEAEA';
export const progressTextColor = '#979797';
export const searchBarTextColor = '#666666';

// cores secundarias
export const vermelhoRSV = '#A00000';
export const verde = '#22372B';
export const neutroQuente1 = '#D7CFCA';
export const offWhite = '#D6D1C4';
export const offWhite15 = '#F9F8F6';
export const whiteSecondary = '#FAFAFA';
export const neutroFrio1 = '#D8D9DA';
export const leve = '#F9E1DD';

// cores do sistema
export const vermelhoAlerta = '#EF1E1E';
export const verdeSucesso = '#38A238';
export const amareloAtencao = '#FEA528';
export const dropDownBorderColor = '#DDDDDD';

// larguras de borda
export const borderWidthNone = 0;
export const borderWidthHairline = 1;
export const borderWidthThin = 2;
export const borderWidthThick = 4;
export const borderWidthHeavy = 8;
export const borderDiscountTag = 12;

// cores de borda
export const borderButton = '#333333';
export const borderColorProduct = '#757575';

// backgrounds
export const backgroundMenuOpened = '#F1F1F1';
export const backgroundApp = '#F9F9F9';
export const modalBackDropColor = '#AFAFAF';
export const backgoundInput = '#F0F0F0';
export const backgoundDivider = '#F5F5F5';
