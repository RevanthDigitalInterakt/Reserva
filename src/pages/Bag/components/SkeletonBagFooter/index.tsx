import { Box } from '@usereservaapp/reserva-ui';
import React from 'react';

export default function SkeletonBagFooter() {
  return (
    <Box px="xxs" testID="com.usereserva:id/skeletonBagFooter_bag">
      <Box>
        <Box
          flexDirection="row"
          justifyContent="space-between"
          marginTop="micro"
        >
          <Box>
            <Box bg="neutroFrio1" width={63} height={22} />
            <Box
              bg="neutroFrio1"
              width={95}
              height={22}
              marginTop="nano"
            />
          </Box>

          <Box alignItems="flex-end">
            <Box bg="neutroFrio1" width={63} height={22} />
            <Box
              bg="neutroFrio1"
              width={95}
              height={22}
              marginTop="nano"
            />
          </Box>
        </Box>

        <Box
          bg="neutroFrio1"
          width="100%"
          height={50}
          marginTop="micro"
        />
      </Box>
    </Box>
  );
}
