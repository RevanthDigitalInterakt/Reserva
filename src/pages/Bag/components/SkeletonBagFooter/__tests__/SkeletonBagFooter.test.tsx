import { render, screen } from '@testing-library/react-native';
import React from 'react';
import { ThemeProvider } from 'styled-components/native';

import SkeletonBagFooter from '..';
import { theme } from '../../../../../base/usereservappLegacy/theme';

const TestingComponent = (
  <ThemeProvider theme={theme}>
    <SkeletonBagFooter />
  </ThemeProvider>
);

describe('SkeletonBagFooter', () => {
  beforeEach(() => {
    render(TestingComponent);
  });

  it('should render without error and match to snapshot', () => {
    expect(screen.getByTestId('com.usereserva:id/skeletonBagFooter_bag')).toBeVisible();
    expect(screen.toJSON()).toMatchSnapshot();
  });
});
