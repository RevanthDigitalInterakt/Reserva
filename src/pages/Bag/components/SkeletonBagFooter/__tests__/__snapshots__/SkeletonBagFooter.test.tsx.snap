// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonBagFooter should render without error and match to snapshot 1`] = `
<View
  px="xxs"
  style={
    {
      "paddingLeft": 24,
      "paddingRight": 24,
    }
  }
  testID="com.usereserva:id/skeletonBagFooter_bag"
>
  <View
    style={{}}
  >
    <View
      flexDirection="row"
      justifyContent="space-between"
      marginTop="micro"
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginTop": 12,
        }
      }
    >
      <View
        style={{}}
      >
        <View
          bg="neutroFrio1"
          height={22}
          style={
            {
              "backgroundColor": "#D8D9DA",
              "height": 22,
              "width": 63,
            }
          }
          width={63}
        />
        <View
          bg="neutroFrio1"
          height={22}
          marginTop="nano"
          style={
            {
              "backgroundColor": "#D8D9DA",
              "height": 22,
              "marginTop": 8,
              "width": 95,
            }
          }
          width={95}
        />
      </View>
      <View
        alignItems="flex-end"
        style={
          {
            "alignItems": "flex-end",
          }
        }
      >
        <View
          bg="neutroFrio1"
          height={22}
          style={
            {
              "backgroundColor": "#D8D9DA",
              "height": 22,
              "width": 63,
            }
          }
          width={63}
        />
        <View
          bg="neutroFrio1"
          height={22}
          marginTop="nano"
          style={
            {
              "backgroundColor": "#D8D9DA",
              "height": 22,
              "marginTop": 8,
              "width": 95,
            }
          }
          width={95}
        />
      </View>
    </View>
    <View
      bg="neutroFrio1"
      height={50}
      marginTop="micro"
      style={
        {
          "backgroundColor": "#D8D9DA",
          "height": 50,
          "marginTop": 12,
          "width": "100%",
        }
      }
      width="100%"
    />
  </View>
</View>
`;
